### ISSUE TITLE
boot.tmpOnTmpfs broken by recent nixos-unstable update

### ISSUE THREAD

**Issue by peti** [NixOS/nixpkgs#109609]
After updating `nixos-unstable` to 257cbbcd3ab, my system no longer boots correctly. The issue appears to be

    boot.tmpOnTmpfs = true;

in `configuration.nix`. This is what syslog has to say:

~~~~
systemd[1]: Mounting /tmp...
mount[1428]: mount: /tmp: special device tmpfs does not exist.
systemd[1]: tmp.mount: Mount process exited, code=exited, status=32/n/a
systemd[1]: tmp.mount: Failed with result 'exit-code'.
systemd[1]: Failed to mount /tmp.
~~~~~

So apparently the init ram disk has lost the ability to use the `tmpfs` file system?

**Comment by tu-maurice** [comment]
Seems like there is already a fix for that: ca7b35d2d9c276cf08d9ca8248eae004f4e9d295

Have you tried another update of nixos-unstable?

**Comment by andersk** [comment]
Closing as a duplicate of #109118.
